### ISSUE TITLE
TypedDict is not supported

### ISSUE THREAD

**Issue by ahuang11** [shroominic/funcchain#30]
Would it be possible to support it to avoid pydantic dependency?

**Comment by shroominic** [comment]
Yeah need to experiment with that should def be possible but not sure yet how to best do it.
Maybe converting it in the background into a pydantic model to utilise the already exsisting parsers and converting it into the dict again.
